src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
-body {
-  margin: 0;
+@import './reset';
+@import './utils/mixins';
+
+.wrapper {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  box-sizing: initial;
+
+  &__minutes {
+    @include hand(20vmin, 3vmin, #0700ff);
+
+    transform: translate(-50%, -10vmin) rotate(-90deg);
+    animation-name: hand-move;
+    animation-duration: 3600s;
+    animation-iteration-count: infinite;
+    animation-timing-function: steps(60);
+  }
+
+  &__seconds {
+    @include hand(38vmin, 1.5vmin, #2c8000);
+
+    transform: translate(-50%, -19vmin) rotate(-90deg);
```

Comment:
why did you use -19vmin, may be better -50%?

Reply:
you can try)
the result may surprise)
first I put this property inside mixin. I don't know how I realized that I should shift a half distance from a width and throw away additional property)
but then I noticed that the center element looks bad and needs to be a little bit lower. So I took the property and passed it inside class. But -50% did not help me) -1200% looked better)))